dotCMS/src/main/java/com/dotcms/publisher/endpoint/bean/factory/PublishingEndPointFactory.java

```diff
@@ -18,10 +18,9 @@
 public class PublishingEndPointFactory {
 
     public PublishingEndPoint getPublishingEndPoint(final String protocol) {
-        if (!UtilMethods.isSet(protocol)) {
-            return null;
-        } else if (PushPublisher.PROTOCOL_HTTP.equalsIgnoreCase(protocol)
-                || PushPublisher.PROTOCOL_HTTPS.equalsIgnoreCase(protocol)) {
+        if (!UtilMethods.isSet(protocol)
```

Comment:
I think the first if is the default one, check if it can be rewrite such as:
```
if (AWSS3Publisher.PROTOCOL_AWS_S3.equalsIgnoreCase(protocol)) {
            return new AWSS3PublishingEndPoint();
        } else if (StaticPublisher.PROTOCOL_STATIC.equalsIgnoreCase(protocol)) {
            return new StaticPublishingEndPoint();
        }
        
       return PushPublishingEndPoint(); // not protocol, PROTOCOL_HTTP, PROTOCOL_HTTPS